eras/conway/impl/cardano-ledger-conway.cabal

```diff
@@ -116,6 +116,7 @@ library testlib
         Test.Cardano.Ledger.Conway.Imp.EpochSpec
         Test.Cardano.Ledger.Conway.Imp.EnactSpec
         Test.Cardano.Ledger.Conway.Imp.GovSpec
+        Test.Cardano.Ledger.Conway.Imp.GovCertSpec
```

Comment:
You do need to actually make sure that the tests from the new module are actually executed, but [there is another PR by Joosep](https://github.com/IntersectMBO/cardano-ledger/pull/4162/files#diff-b8a51c5e12091af98adea02df5db2097daa3efcd96dcaa7bd9a5d7be38f62299) that already takes care of it, so we can merge this one as is. I am just letting you know for the future

Reply:
Oops! I forgot! Sorry!